src/aws-cli/aws_zsh_completer.sh

```diff
@@ -0,0 +1,60 @@
+# Source this file to activate auto completion for zsh using the bash
```

Comment:
Is this file published anywhere by a trusted source? Can you point me to it?

Reply:
It's in [aws-cli sources](https://github.com/aws/aws-cli/blob/v2/bin/aws_zsh_completer.sh) but it's not included with the installer.

Reply:
Thanks! As we are copying this script file, it's very likely to get stale in long run. 

Is it possible to add an action similar to https://github.com/devcontainers/features/blob/main/.github/workflows/update-dotnet-install-script.yml which syncs this file? Thanks!